packages/grafana-ui/src/components/Logs/LogRowMessage.tsx

```diff
@@ -235,38 +105,15 @@ class UnThemedLogRowMessage extends PureComponent<Props, State> {
             />
           )}
           <span className={cx(styles.positionRelative, { [styles.rowWithContext]: showContext })}>
-            {parsed && (
-              <Highlighter
-                style={{ whiteSpace: 'pre-wrap' }}
-                autoEscape
-                highlightTag={FieldHighlight(this.onClickHighlight)}
-                textToHighlight={entry}
-                searchWords={parsedFieldHighlights}
-                highlightClassName={cx([style.logsRowFieldHighLight])}
-              />
-            )}
-            {!parsed && needsHighlighter && (
-              <Highlighter
-                style={{ whiteSpace: 'pre-wrap' }}
-                textToHighlight={entry}
-                searchWords={highlights}
-                findChunks={findHighlightChunksInText}
-                highlightClassName={highlightClassName}
-              />
-            )}
-            {hasAnsi && !parsed && !needsHighlighter && <LogMessageAnsi value={raw} />}
-            {!hasAnsi && !parsed && !needsHighlighter && entry}
-            {showFieldStats && (
-              <div className={cx([style.logsRowStats])}>
-                <LogLabelStats
-                  stats={fieldStats!}
-                  label={fieldLabel!}
-                  value={fieldValue!}
-                  onClickClose={this.onClickClose}
-                  rowCount={fieldCount}
-                />
-              </div>
-            )}
+            <Highlighter
+              style={{ whiteSpace: 'pre-wrap' }}
```

Comment:
This style should be extracted to a constant.

Reply:
👍done!